### PR TITLE
Hw 3

### PR DESCRIPTION
**Pull request by viskosa**

### CODE REVIEW

File: js-core/homeworks/homework-3/src/main.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+
+/* @@ SUPER !----------------------------------------------------------------
+  Напишите функцию, которая будет
+   возвращать 'Fizz' если передаваемый параметр кратен 3,
+   'Buzz', если передаваемый параметр кратен 5,
+   'FizzBuzz' - если параметер кратен 3 и 5
+   Если передаваемое число не кратно 3 или 5, то вернуть указанный параметр
+*/
+//the first solution----------------------------------
```
Comment (OlegLustenko) [inline js-core/homeworks/homework-3/src/main.js:129]: you have a lot of cool solution. 

But you have to return value from fizzBuzz function, and then you can alert or log it, or something else

Comment (OlegLustenko) [reply js-core/homeworks/homework-3/src/main.js:129]: When you do that way you could think about simplifying your conditionals

Comment (viskosa) [reply js-core/homeworks/homework-3/src/main.js:129]: Do you mean something like that? See the second solution, please

Comment (viskosa) [reply js-core/homeworks/homework-3/src/main.js:129]: Unfortunatelly, I usually have problems with simplifying my code (

**Review by OlegLustenko — APPROVED** [review]

File: js-core/homeworks/homework-3/src/main.js (hunk 1 above)
Comment (OlegLustenko) [reply js-core/homeworks/homework-3/src/main.js:129]: @viskosa don't worry I could help you with simplifying, just DM me if you have some questions

**Review by OlegLustenko — APPROVED** [review]

**Pull request merged**